### PR TITLE
Wizard component

### PR DESCRIPTION
**Pull request by Robinnnnn**
Addresses #95 

https://user-images.githubusercontent.com/12162433/115134915-71774800-9fe2-11eb-8b7e-477c69355bb3.mov

### CODE REVIEW

File: src/components/core/Text/Text.tsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ export const Title = styled.p`
 `;
 
 type TextProps = {
-  light?: boolean;
+  color?: colors;
```
Comment (kaitoo1) [inline src/components/core/Text/Text.tsx:12]: oo nice call to add this as an option

File: src/scenes/CreateCollection/CreateCollection.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import OrganizeCollections from './steps/OrganizeCollections';
+
+function CreateCollection(_: RouteComponentProps) {
+  return (
+    <Wizard
+      render={({ step, next, previous }) => {
+        return (
+          <>
+            <Steps>
+              <Step id="create" render={FirstCollection} />
```
Comment (kaitoo1) [inline src/scenes/CreateCollection/CreateCollection.tsx:16]: maybe `FirstCollection` can be named `CreateFirstCollection` so all the steps in the wizard have a verb and it's explicit what the intended action is

**Review by kaitoo1 — APPROVED** [review]
approved with minor comment about step naming convention

File: src/scenes/CreateCollection/CreateCollection.tsx (hunk 2 above)
Comment (Robinnnnn) [reply src/scenes/CreateCollection/CreateCollection.tsx:16]: great idea on naming convention- will make the fix and merge!

**Pull request merged**